### PR TITLE
Cleanup README

### PR DESCRIPTION
**Pull request by blankenshipz**
* Remove install instruction for docker (let's just install nib)
* Update the Usage / nib sections with philosophy/detail information

:eyes: @johnallen3d

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,51 +29,6 @@ For additional information and a list of [supported commands](./docs/commands.md
...
-    In order to point your docker client at the new machine you'll need to set some environment variables.
-
-    ```bash
-    eval $(dinghy shellinit)
-    ```
-
-    This will need to be executed per shell session. If you're using docker regularly you should consider adding this to your profile file of choice (`.bashrc`, `.zshrc` etc).
-
-1. Copy the alias below into your shell configuration and provide a value for "YOUR_GIT_KEY" (this will allow the `nib` tool to access git repos on your behalf)
 1. The most convenient way to use nib is by creating an alias or shell function. Here is an alias you can add to your profile that will make `nib` appear as a command. Note, in order for the `nib update` command (download the latest version) to work
```
Comment (johnallen3d) [inline README.md]: Looks like this slipped through the crack again:

> Note, in order for the `nib update` command (download the latest version) to work

**Comment by johnallen3d** [comment]
Other than the one comment above I think this is a great! 👍

**Comment by blankenshipz** [comment]
@johnallen3d :eyes:

**Comment by johnallen3d** [comment]
:shipit:

**Pull request merged**